python/cudf/cudf/tests/test_parquet.py

```diff
@@ -20,6 +20,36 @@ def datadir(datadir):
     return datadir / "parquet"
 
 
+@pytest.fixture(params=[1, 5, 10, 100])
+def simple_pdf(request):
+    types = ["int8", "int16", "int32"]
```

Comment:
can we add more types here? Notably `int64`, `float32`, `float64`, `string`, and possible some `datetime` types with different resolutions?